src/paperless_tesseract/tests/test_date.py

```diff
@@ -115,6 +115,42 @@ def test_date_format_7(self):
             datetime.datetime(2019, 3, 1, 0, 0, tzinfo=tz.tzutc())
         )
 
+    @mock.patch(
+        "paperless_tesseract.parsers.RasterisedDocumentParser.SCRATCH",
+        SAMPLE_FILES
+    )
+    def test_date_format_8(self):
+        input_file = os.path.join(self.SAMPLE_FILES, "")
+        document = RasterisedDocumentParser(input_file)
+        document.TEXT_CACHE = ("lorem ipsum\n"
```

Comment:
The tests are failing 'cause I changed `.TEXT_CACHE` to `._text` in `RasterisedDocumentParser` in keeping with [pep8's rule about constants](https://pep8.org/#constants).

Reply:
done